user/enterprise/setting-up-travis-ci-enterprise.md

```diff
@@ -0,0 +1,162 @@
+RAM and 8 CPUs**.
+
+If you're running on EC2, we recommend the **c4.2xlarge** instance type for both **Platform** and **Worker**. We also recommend using an image that uses EBS for the root volume, as well as allocating 40 gigs of space to it. It is also recommended _not_ to destroy the volume on instance termination.
+
+For [high availability (HA)](/user/enterprise/high-availability/) configurations, you will also need:
+
+* [Redis](https://redis.io/),
+* [RabbitMQ](https://www.rabbitmq.com/)
+* [Postgres](https://www.postgresql.org/)
+
+You can also try services like [compose.com](https://compose.com/) if you would like these services hosted outside your organization.
+
+## 1. Setting up  Enterprise Platform virtual machine
+
+The Travis CI Enterprise Platform handles licensing, coordinates worker
+processes, and maintains the Enterprise user and admin dashboard. It must be
+installed on it's own machine instance, separate from that of the Travis CI
+Enterprise worker. We recommend using AWS' `c4.2xlarge` instance running
+Ubuntu 16.04 LTS as the underlying operating system.
+
+1. *On your virtual machine management platform*, create a Travis CI Platform Security Group.
+
+    If you're setting up your AMI for the first time you need to create
+    a Security Group. From the EC2 management console, create an entry for
+    each port in the table below:
+
+    | Port | Service         | Description                                                                  |
+    |:-----|:----------------|:-----------------------------------------------------------------------------|
+    | 8800 | Custom TCP Rule | This port is to access the admin dashboard for your Enterprise installation. |
+    | 5672 | Custom TCP Rule | For RabbitMQ Non-SSL.                                                        |
+    | 4567 | Custom TCP Rule | For RabbitMQ SSL.                                                            |
+    | 443  | HTTPS           | Web application over HTTPS access.                                           |
+    | 80   | HTTP            | Web application access.                                                      |
+    | 22   | SSH             | SSH access.                                                                  |
+
+1. If you're using a hostname and not just an IP address, this would be a time to configure it.
+
+1. *On your new virtual machine*, download and run the installation script:
+
+
+    ```
+    curl -sSL -o /tmp/installer.sh https://enterprise.travis-ci.com/install
+    sudo bash /tmp/installer.sh
+    ```
+
+3. *In your browser*, navigate to `https://<hostname>:8800` (your Enterprise
+installation's hostname, port 8800) to complete the setup:
+
+    a. Add a secure certificate / configure a trusted one [HTTPS Replicated page]  
+    b. Upload your license - [Validaing license file]
+    c. Configure access to the Admin Console (password / openldap)
```

Comment:
Should we point out here that LDAP / Active Directory only applies to the management console and not to Travis Enterprise as a whole? 

If people want LDAP / Active Directory authentication for end users then they only need to configure this in GHE where this then automatically applies to Travis Enterprise as well.